src/main/kotlin/uk/gov/dluhc/notificationsapi/mapper/RejectedDocumentsMapper.kt

```diff
@@ -10,15 +11,19 @@ class RejectedDocumentsMapper(
     private val rejectedDocumentReasonMapper: RejectedDocumentReasonMapper,
     private val rejectedDocumentTypeMapper: RejectedDocumentTypeMapper
 ) {
+    private val subItemSeparator = "\n  * "
```

Comment:
Is this correct for both emails and letters?

Reply:
I believe so https://www.notifications.service.gov.uk/using-notify/formatting#bullets

Planning to check the nested bullet point formatting once these changes are on int though